fixtures/generate.js

```diff
@@ -0,0 +1,98 @@
+import fs from 'fs';
+import path from 'path';
+import config from '../yeep.config';
+import { generateFakeUser } from './models/users';
+import { generateFakeOrg } from './models/orgs';
+import { generateFakePermissions } from './models/permissions';
+import { generateFakeRoles } from './models/roles';
+import { promisify } from 'util';
+
+const writeFileAsync = promisify(fs.writeFile);
+
+const generateFakeUsers = (numberOfUsers) => {
```

Comment:
Might be worth renaming to `generateManyFakeUsers` in order to highlight the difference from `generateFakeUser`. I leave it up to you - merely just a suggestion.

If you do decide to go ahead with this please update the methods below accordingly.